Emily's first pass

emilyannmitchell opened the pull request:
Hi- I did a first pass through here. I think after reading through this. There are numerous comments, set off with <!--EM. Please review/respond. 

One thing that I think we need to do is to take out Step 1 in this piece and create a separate piece that all of our RHM content points to about Configuring an OpenShift Cluster on RHM. The content is identical to the first steps here: https://developer.ibm.com/tutorials/get-started-using-a-cockroachdb-operator-hosted-on-red-hat-marketplace/ I think it might be better to just make it separate piece so that we can point to it in every piece of contetn without hurting our SEO (which would happen if we have the same content in the top section of each piece.

emilyannmitchell committed:
Emily's first pass

Hi- I did a first pass through here. I think after reading through this. There are numerous comments, set off with <!--EM. Please review/respond. 

One thing that I think we need to do is to take out Step 1 in this piece and create a separate piece that all of our RHM content points to about Configuring an OpenShift Cluster on RHM. The content is identical to the first steps here: https://developer.ibm.com/tutorials/get-started-using-a-cockroachdb-operator-hosted-on-red-hat-marketplace/ I think it might be better to just make it separate piece so that we can point to it in every piece of contetn without hurting our SEO (which would happen if we have the same content in the top section of each piece.

RahulReddyRavipally commented:
Hi Emily

Step 1 and 2 in this tutorial https://developer.ibm.com/tutorials/get-started-using-a-cockroachdb-operator-hosted-on-red-hat-marketplace/ will be common for all the operators. We can make a separate tutorial/how-to for step 1 and step 2 so that all our RHM assets can refer to this content.

Thank You

RahulReddyRavipally commented on README.md:
```diff
@@ -12,87 +11,50 @@ authors:
+Red Hat® OpenShift® 4 users can access certified software for container-based environments via the [Red Hat Marketplace](https://marketplace.redhat.com/en-us/about). Software in the marketplace is immediately available to deploy on any Red Hat OpenShift cluster in a fast, integrated way. Follow the steps in this tutorial to set up and deploy a CrunchyDB operator hosted on Red Hat Marketplace. <!- ...
 
-type: tutorial
+<!--EM: Are there any tech prereqs we need to mention here? It seems like having an OpenShift 4 license? Or setting up a cluster?-->
```
No tech prereqs here.

RahulReddyRavipally commented on README.md:
```diff
@@ -154,98 +116,109 @@ type: tutorial
-To login you would require token which can be genrated after you login to Openshift Cluster web console. See below screenshot to `copy the path`.
-![](doc/source/images/Login-CopyCommand.png)
+### Step 4: Create and deploy a CrunchyDB Operator on an OpenShift Cluster and create a database
+<!--EM: In this section, what are `pgo` and pvc`-->
```
These are libraries that are used in terminal to Connect Connect to the Openshift Cluster

RahulReddyRavipally commented on README.md:
```diff
@@ -154,98 +116,109 @@ type: tutorial
+    a. `` pgo show cluster -n pgo hippo``
     b.  ``pgo test -n pgo hippo``
+    
+    <!--EM: What are those different commands for? Or what do they do when you run them?-->
```
This is to check if to validate if the connection is successful or not

emilyannmitchell committed:
Emily's second pass

I took out the duplicate content and pointed to the tutorial in a prereqs section. Do you think that's enough or do you think I need to add it as Step 1. "Configure a Red Hat OpenShift cluster on Red Hat Marketplace" and the say: The instructions to do this are in this tutorial: [link to the tutorial].

A few other questions:
1. This line says, "check the logs and be sure there are no errors in the Ansible script" . . . does the reader need to have Ansible installed or does that happen automatically with an OpenShift install?
2. Under Step 3. Access the cluster on your localhost, you mentioned pgAdmin 4 console . . . . what is pgAdmin 4 console? Is that part of the marketplace? how does a reader access that?

emilyannmitchell committed:
Merge branch 'master' into patch-1

RahulReddyRavipally commented on README.md:
```diff
@@ -132,42 +74,28 @@ Red Hat® OpenShift® 4 users can access certified software for container-based
 
 1. Use the new namespace where you installed the Crunchy Postgres operator.
 
 1. Run the following command in your command line interface (CLI). Once it runs successfully, check the logs and be sure there are no errors in the Ansible script. <!--EM: Do they have to have Ansible installed?-->. Wait for the pod state to change to Complete.
```
Ansible is already present in every system. Users need not install this explicitly.

RahulReddyRavipally commented on README.md:
```diff
@@ -181,20 +109,19 @@ This will create a database (pods) in the pgo namespace.
+### Step 3: Access the cluster on your localhost
 
-T view the results of the commands you ran in the earlier steps via the pgAdmin 4 console<!--EM: What is pgAdmin 4 console? Is that the RH console?-->. Access the console at localhost with port forwarding.
+To view the results of the commands you ran in the earlier steps via the pgAdmin 4 console<!--EM: What is pgAdmin 4 console? Is that the RH console?-->. Access the console at localhost with port forwarding.
```
pgAdmin console is crunchydb console, not the RH console.

emilyannmitchell committed:
Update README.md
